Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/Log.java

```diff
@@ -0,0 +1,45 @@
+package org.dolphinemu.dolphinemu.utils;
+
+
+import org.dolphinemu.dolphinemu.BuildConfig;
+
+public class Log
```

Comment:
final class

Constructor should also be private if it's a purely static class to stop instantiation.

Also documentation should be provided on the class (methods are self-explanatory) why this class is to be used instead of the android logging class.